install_config/oab_broker_configuration.adoc

```diff
@@ -0,0 +1,674 @@
+is stored as a ConfigMap object in the broker's namespace
+(*openshift-ansible-service-broker* by default).
+
+.Example OpenShift Ansible Broker Configuration File
+[source,yaml]
+----
+registry: <1>
+  - type: dockerhub
+    name: docker
+    url: https://registry.hub.docker.com
+    org: <dockerhub_org>
+    fail_on_error: false
+  - type: rhcc
+    name: rhcc
+    url: https://registry.access.redhat.com
+    fail_on_error: true
+    white_list:
+      - "^foo.*-apb$"
+      - ".*-apb$"
+    black_list:
+      - "bar.*-apb$"
+      - "^my-apb$"
+  - type: local_openshift
+    name: lo
+    namespaces:
+      - openshift
+    white_list:
+      - ".*-apb$"
+dao: <2>
+  etcd_host: localhost
+  etcd_port: 2379
+log: <3>
+  logfile: /var/log/ansible-service-broker/asb.log
+  stdout: true
+  level: debug
+  color: true
+openshift: <4> 
+  host: ""
+  ca_file: ""
+  bearer_token_file: ""
+  image_pull_policy: IfNotPresent
+  sandbox_role: "edit"
+  keep_namespace: false
+  keep_namespace_on_error: true
+broker: <5>
+  bootstrap_on_startup: true
+  dev_broker: true
+  launch_apb_on_bind: false
+  recovery: true
+  output_request: true
+  ssl_cert_key: /path/to/key
+  ssl_cert: /path/to/cert
+  refresh_interval: "600s"
+  auth:
+    - type: basic
+      enabled: true
+secrets: <6>
+  - title: Database credentials
+    secret: db_creds
+    apb_name: dh-rhscl-postgresql-apb
+----
+<1> See xref:oab-broker-config-registry[Registry Configuration] for details.
+<2> See xref:oab-broker-config-dao[DAO Configuration] for details.
+<3> See xref:oab-broker-config-log[Log Configuration] for details.
+<4> See xref:oab-broker-config-openshift[OpenShift Configuration] for details.
+<5> See xref:oab-broker-config-broker[Broker Configuration] for details.
+<6> See xref:oab-broker-config-secrets[Secrets Configuration] for details.
+
+[[install-config-oab-modifying]]
+== Modifying the OpenShift Ansible Broker Configuration
+
+To modify the OAB's default broker configuration after it has been deployed:
+
+. Edit the the *broker-config* ConfigMap object in the OAB's namespace as a user
+with *cluster-admin* privileges:
++
+----
+$ oc edit configmap broker-config -n openshift-ansible-service-broker
+----
+
+. After saving any updates, redeploy the OAB's deployment configuration for the
+changes to take effect:
++
+----
+$ oc rollout latest dc/asb -n openshift-ansible-service-broker
+----
+
+[[oab-broker-config-registry]]
+== Registry Configuration
+
+The `registry` section allows you to define the registries that the broker should look at
+for APBs.
+
+.`registry` Section Configuration Options
+[options="header",cols="1,3,1"]
+|===
+
+|Field |Description |Required
+
+|`name`
+|The name of the registry. Used by the broker to identify APBs from this registry.
+|Y
+
+|`auth_type`
+|How the broker should read the registry credentials. Can be `config` (uses
+`user` and `pass` as defined in the `registry` section) or `secret` (uses a secret
+in the broker namespace).
```

Comment:
@jianlinliu Are you referring to the instructions from the comment in https://bugzilla.redhat.com/show_bug.cgi?id=1539310#c2 ? Because I had removed the `file` type instructions until that BZ was resolved.

Reply:
From your PR merge requestion, I think this doc will be released in version 3.9. If so, that bug will be fixed until then. So, I think we should add the `file` type instruction.